doc/tutorial/01_introduction/index.md

```diff
@@ -8,7 +8,7 @@ of what to expect in the other tutorials and how to use this documentation.
 \tutorial_head{Easy, 30 min, \ref setup, [Ranges](https://github.com/seqan/seqan3/wiki/Ranges)\,
```

Comment:
The ranges entry is a bit old. There is https://en.cppreference.com/w/cpp/ranges, but it doesn't explain much.
Either we find a good reference, or we update the wiki a little bit. The wiki actually looks good, but we need to remove the mentions of range-v3 and cpp17+concepts as they are outdated.

Reply:
I deleted both references as a hello world tutorial should not require you to know about concepts nor ranges.